nilearn/datasets/func.py

```diff
@@ -60,6 +60,17 @@ def fetch_haxby(
 ):
     """Download and loads complete haxby dataset.
 
+    Notes
```

Comment:
I think there are 2 Notes section here:

this is causing a doc build failure: 

`ValueError: The section Notes appears twice in  Download and loads complete haxby dataset.`

https://github.com/nilearn/nilearn/actions/runs/17814603178/job/50687895141?pr=5690#step:7:281